websites/D/Deezer/presence.ts

```diff
@@ -34,88 +75,104 @@ presence.on("UpdateData", async () => {
           : false;
 
     if (!show) {
-      (title = document.querySelector(".track-link:nth-child(1)").textContent),
-        (author = document.querySelector(".track-link:nth-child(2)")
-          .textContent);
+      title = document.querySelector(".track-link:nth-child(1)").textContent;
+      artist = document.querySelector(".track-link:nth-child(2)").textContent;
+          presenceData.details = title,
+          presenceData.state = artist,
+          presenceData.largeImageKey = "deezer",
+          presenceData.smallImageKey = paused ? "pause" : "play",
+          presenceData.smallImageText = paused ? (await strings).pause : (await strings).play,
+          presenceData.startTimestamp = timestamps[0],
```

Comment:
make sure this is correct and working because of the , instead of ;

Reply:
Yep it does work correctly with the , instead of ; but changing it would do no harm. Will change.